### ISSUE TITLE
Request for a new release tag

### ISSUE THREAD

**Issue by ilovezfs** [relaxng/jing-trang#212]
For Homebrew, we're having some trouble building the most recent actual tag, but HEAD seems to be OK. Since we only distribute tags, do you mind tagging a new version so that we can get the formula working again? Thanks so much!

Cf. https://github.com/Homebrew/homebrew-core/pull/4312

**Comment by gambl** [comment]
A new version has been tagged (V20151127) based upon the current HEAD. Hurray!

**Comment by sideshowbarker** [comment]
> A new version has been tagged (V20151127) based upon the current HEAD. Hurray!

Yup, https://github.com/relaxng/jing-trang/releases/tag/V20151127

If there’s anything else needed before I close this issue, lemme know

**Note:** I deleted the initial V20151127 tag I created yesterday because I forgot to change the version number in the `version.xml` file to match. But I subsequently updated the `version.xml` file and then remade the V20151127 tag so that `version.xml` change would get included in it. So anybody who happened to do anything with the contents from state of the V20151127 tag yesterday should now today re-do whatever they did (in order to get the `version.xml` change).
